releases/release-1.23/README.md

```diff
@@ -38,6 +38,7 @@ description: |
 The 1.23 release cycle is proposed as follows:
 
 - **Monday, August 23, 2021**: Week 1 - Release cycle begins
+- **Thursday, September 2, 2021**: Week 2 - [Production Readiness Soft Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
```

Comment:
```suggestion
- **Thursday, September 2, 2021**: Week 2 - [Production Readiness Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
```
I suggest not to differentiate between a soft freeze and a regular freeze. An exception to the PRR freeze can be part of an [exception request](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md)

Reply:
We do not want to require an exception request for folks being late with this. We don't need another hard deadline that requires exception requests - I think that would be overly burdensome.